src/common/translation/i18n/en.json

```diff
@@ -330,6 +330,20 @@
     "searchCategories": "Browse Categories",
     "searchEvents": "Find something to do"
   },
+  "meta":{
+    "events": {
+      "description": "Tapahtumat.hel.fi’s event search. Concerts, gigs, theatre performances, exhibitions, culture, activities and other events in Helsinki."
+    },
+    "courses": {
+      "description": "Tapahtumat.hel.fi’s activity search. Activities, courses, clubs and camps for children and young people in Helsinki."
+    },
+    "collections": {
+      "description": "Check out our recommendations on the most interesting upcoming events."
```

Comment:
There supposed to be Exclamation mark `!` in the end of sentence 
<img width="597" alt="Screenshot 2021-03-26 at 12 59 38 PM" src="https://user-images.githubusercontent.com/7648194/112622163-524a2800-8e33-11eb-841f-d16e5145af81.png">

Reply:
Funny enough the finnish and swedish translations do not have exclamation mark, so not sure about that either :)

Reply:
i was thinking it was a typo, as it was only in one language. i will confirm and fix in case